Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -990,7 +990,7 @@ def file_acquisition_package_request(self, acquisition_id):
         response = self._http_request(
             method='GET',
             url_suffix=f'acqs/files/{acquisition_id}.zip',
-            headers=headers,
+            headers=self._headers | headers,
```

Comment:
please put a short comment regarding the need of both 
the new Accept and the token form the self._headers